lib/omniauth/strategies/openid_connect.rb

```diff
@@ -170,6 +171,11 @@ def authorize_uri
           hd: options.hd,
           acr_values: options.acr_values,
         }
+
+        if options.extra_authorize_params
```

Comment:
I guess this check will always be `true` since you have empty hash as a default value. It would make sense to check `unless options.extra_authorize_params.empty?`.

Please correct me if I'm wrong.

Reply:
ah, good catch! I'll fix this up :)